pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -84,7 +84,7 @@ func (d *DependenciesDistributor) Start(ctx context.Context) error {
 	d.stopCh = ctx.Done()
 
 	bindingWorkerOptions := util.Options{
-		Name:          "resourceBinding reconciler",
+		Name:          "dependencies distributor resourceBinding reconciler",
```

Comment:
The name has been changed to [dependencies-distributor](https://github.com/karmada-io/karmada/blob/82fc37ea04aca442f378a013e8846104595037ea/pkg/dependenciesdistributor/dependencies_distributor.go#L66) by #5799, and released in v1.12.0.
https://github.com/karmada-io/karmada/blame/82fc37ea04aca442f378a013e8846104595037ea/pkg/dependenciesdistributor/dependencies_distributor.go#L66